### ISSUE TITLE
macOS Big Sur v11.2 (M1/Apple Silicon): pip installation issues

### ISSUE THREAD

**Issue by eliheuer** [googlefonts/fontbakery#3234]
## User Environment
```
macOS Big Sur v11.2 (M1/Apple Silicon)
Python v3.9.2
ttfautohint v1.8.3 (in $PATH)
pip v21.0.1
```
## Observed Behaviour
I'm trying to install fontbakery on an M1(Apple Silicon)/Big Sur 11.2 Mac using `pip install --editable .`, but I get the following error:
```
$ pip install --editable .
...

ERROR: Could not find a version that satisfies the requirement ttfautohint-py (from fontbakery)
ERROR: No matching distribution found for ttfautohint-py
```
This is similar to the issue Dave reported recently: https://github.com/googlefonts/fontbakery/issues/3229

However, there is one difference. If I just try to install with `pip install fontbakery`, it installs, but I get `fontbakery-0.5.0` instead of `fontbakery-0.7.34` which is the newest version on PyPI. I've never had that issue with pip before, so I worry some people might use an outdated version of fontbakery on M1 Macs without realizing it?

## Expected Behaviour

I want to install FontBakery on an Apple Silicon Mac running Big Sur macOS v11. Ideally using `pip install --editable .`.

**Comment by chrissimpkins** [comment]
Is ttfautohint-py used anywhere other than the condition that returns file sizes for ttf with and without instruction sets?

https://github.com/googlefonts/fontbakery/blob/d1e55b8f29fff9782231d18192f194c82cd8a192/Lib/fontbakery/profiles/googlefonts_conditions.py#L279-L328

**Comment by felipesanches** [comment]
fixed at #3271
